tests/ops/integration_tests/saas/test_auth0_task.py

```diff
@@ -0,0 +1,165 @@
+import random
+
+import pytest
+import requests
+
+from fidesops.core.config import config
+from fidesops.graph.graph import DatasetGraph
+from fidesops.models.privacy_request import PrivacyRequest
+from fidesops.schemas.redis_cache import PrivacyRequestIdentity
+from fidesops.task import graph_task
+from fidesops.task.graph_task import get_cached_data_for_erasures
+from tests.ops.graph.graph_test_util import assert_rows_match
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_auth0
+def test_auth0_access_request_task(
+    db,
+    policy,
+    auth0_connection_config,
+    auth0_dataset_config,
+    auth0_identity_email,
+) -> None:
+    """Full access request based on the Auth0 SaaS config"""
+    privacy_request = PrivacyRequest(
+        id=f"test_saas_access_request_task_{random.randint(0, 1000)}"
+    )
+    identity = PrivacyRequestIdentity(**{"email": auth0_identity_email})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = auth0_connection_config.get_saas_config().fides_key
+    merged_graph = auth0_dataset_config.get_graph()
+    graph = DatasetGraph(merged_graph)
+
+    v = graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [auth0_connection_config],
+        {"email": auth0_identity_email},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:users"],
+        min_size=1,
+        keys=[
+            "created_at",
+            "email",
+            "email_verified",
+            "identities",
+            "name",
+            "nickname",
+            "picture",
+            "updated_at",
+            "user_id",
+            "last_ip",
+            "last_login",
+            "logins_count",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:user_logs"],
+        min_size=1,
+        keys=[
+            "date",
+            "type",
+            "description",
+            "connection_id",
+            "client_id",
+            "client_name",
+            "ip",
+            "user_agent",
+            "details",
+            "user_id",
+            "user_name",
+            "log_id",
+            "_id",
+            "isMobile",
+            "location_info",
+        ],
+    )
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_auth0
+def test_auth0_erasure_request_task(
+    db,
+    policy,
+    erasure_policy_string_rewrite,
+    auth0_connection_config,
+    auth0_dataset_config,
+    auth0_erasure_identity_email,
+    auth0_erasure_data,
+) -> None:
+    """Full erasure request based on the auth0 SaaS config"""
+
+    privacy_request = PrivacyRequest(
+        id=f"test_saas_erasure_request_task_{random.randint(0, 1000)}"
```

Comment:
Let's change the id to `test_auth0_erasure_request_task` so we can uniquely identify it.